cmd/fleetctl/package.go

```diff
@@ -150,6 +150,30 @@ func packageCommand() *cli.Command {
 				EnvVars:     []string{"FLEETCTL_NATIVE_TOOLING"},
 				Destination: &opt.NativeTooling,
 			},
+			&cli.StringFlag{
+				Name:        "macos-devid-pem-content",
+				Usage:       "Dev ID certificate keypair content in PEM format",
```

Comment:
@lucasmrod this PEM file contains the private/public keypair. An alternative would be to request the PKCS 12 key base64 encoded (as it's a binary file) and the password for the key as well.

This seems less convoluted but happy to change things if you think the other approach is safer.

Reply:
Good question. 

What's the usual representation/storage of the key + certificate?

Reply:
I think both are valid, when you export the certificate from Keychain it's a binary PKCS 12 file, the PEM file is just plain text.

my main concern would be if you have the PEM file in your file system, but as this is intended to be consumed via an env var securely set there might not be a difference, but I'm not an expert on the subject.

Reply:
> I think both are valid, when you export the certificate from Keychain it's a binary PKCS 12 file, the PEM file is just plain text.

But why the change then? Is it because `rcodesign` was not working correctly with the `-p12` arguments?

Reply:
Ah that makes sense!

Maybe we should add docs on how to assemble the keys as exported from the keychain into  `FLEETCTL_MACOS_DEVID_PEM_CONTENT`?

Reply:
yes, absolutely!! 👍 as soon as I get confirmation from Zack that he was able to use the instructions I provided I will document them

Reply:
Sounds good. Apart from the "what", let's also document the "why" somewhere in the code (because as you said, we are currently passing such keypair as environment variables/flags and thus we need encoding)